src/Juvix/Extra/Strings.hs

```diff
@@ -107,6 +107,9 @@ naturalPrint = "natural-print"
 naturalPlus :: IsString s => s
 naturalPlus = "natural-plus"
 
+booleanIf :: IsString s => s
+booleanIf = "boolean-if"
```

Comment:
since we define the type as `Bool`, we should consider `bool-if` instead

Reply:
We use `natural` (and `natural-plus`) for the type we call `Nat` in the standard library. I am using the same pattern for `boolean`. Does that make sense?